src/NodeGraphicsObject.cpp

```diff
@@ -318,13 +318,11 @@ mouseMoveEvent(QGraphicsSceneMouseEvent* event)
 
       oldSize += QSize(diff.x(), diff.y());
 
-      w->setFixedSize(oldSize);
+      w->resize(oldSize);
 
       AbstractNodeGeometry & geometry = nodeScene()->nodeGeometry();
 
-      _proxyWidget->setMinimumSize(oldSize);
-      _proxyWidget->setMaximumSize(oldSize);
-      _proxyWidget->setPos(geometry.widgetPosition(_nodeId));
+      // _proxyWidget->setPos(geometry.widgetPosition(_nodeId));
```

Comment:
Why so? The proxy graphical item must be positioned somehow within the parent item.

Reply:
Oh, I see, it is still done in the function `embedQWidget`